### PR TITLE
4 proposal motivation and purpose

### PR DESCRIPTION
**Pull request by Rachel0619**
- added motivation and purpose to `m1_proposal.md`
- added contents to `README.md`
- added a cover photo (I find this one visually appealing and I'm open to suggestions if you have other images in mind that might better suit our repo!)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,26 @@
-# DSCI-532_2024_20_appname
\ No newline at end of file
+![Project Cover](img/cover.png)
+Surasak Suwanmake//Getty Images
+
+# 🚀🌌📈Hotspot - Visualization of Global Warming and CO2 Emission
+
+## Description
+
+With the consensus among scientists that human activities, especially the emission of carbon dioxide (CO2), are the number one contributor to the global warming, we aim to build an innovative tool to visualize this relationship in an intuitive way. To be more specific, we want to inspect the dynamics between CO2 emissions, global temperature changes, and economy activity indicators (e.g GDP per capita) over the years. By providing a user-friendly interface that allows for investigating the data both globally and at the country level, the dashboard seeks to make the abstract and sometimes overwhelming data on climate change more tangible and understandable.
```
Comment (monazhu) [inline README.md:8]:
```suggestion
Given consensus among scientists that human activities, especially the emission of carbon dioxide (CO2), are the number one contributor to the global warming, we aim to build an innovative tool to visualize this relationship in an intuitive way. To be more specific, we want to inspect the dynamics between CO2 emissions, global temperature changes, and economy activity indicators (e.g GDP per capita) over the years. By providing a user-friendly interface that allows for investigating the data both globally and at the country level, the dashboard seeks to make the abstract and sometimes overwhelming data on climate change more tangible and understandable. 
```

File: reports/m1_proposal.md [hunk 2]
```diff
@@ -2,39 +2,43 @@
...
+
+With the consensus among scientists that human activities, especially the emission of carbon dioxide (CO2), are the number one contributor to the global warming, we aim to build an innovative tool to visualize this relationship in an intuitive way. To be more specific, we want to inspect the intricate dynamics between CO2 emissions, global temperature changes, and economic activity indicators (e.g GDP per capita) over the years. By providing a user-friendly interface that allows for investigating the data both globally and at the country level, the dashboard seeks to make the abstract and sometimes overwhelming data on climate change more tangible and understandable. Going beyond merely presenting statistics, our dashboard offers a compelling narrative on the evolution of CO2 emissions over time. It utilizes a diversified visual tools, including line charts, bar charts, and maps to vividly illustrate the profound impact of human activities on our planet's climate.
```
Comment (monazhu) [inline reports/m1_proposal.md:7]:
```suggestion
Given consensus among scientists that human activities, especially the emission of carbon dioxide (CO2), are the number one contributor to the global warming, we aim to build an innovative tool to visualize this relationship in an intuitive way. To be more specific, we want to inspect the intricate dynamics between CO2 emissions, global temperature changes, and economic activity indicators (e.g GDP per capita) over the years. By providing a user-friendly interface that allows for investigating the data both globally and at the country level, the dashboard seeks to make the abstract and sometimes overwhelming data on climate change more tangible and understandable. Going beyond merely presenting statistics, our dashboard offers a compelling narrative on the evolution of CO2 emissions over time. It utilizes a diversified visual tools, including line charts, bar charts, and maps to vividly illustrate the profound impact of human activities on our planet's climate.
```

File: reports/m1_proposal.md [hunk 3]
```diff
@@ -2,39 +2,43 @@
...
+
+Possible users and stakeholders who would find value in our dashboard include governments, policy makers, scientists and environmental advocacy organizations. Please find more detailed discussion of the use cases in section 3.
```
Comment (monazhu) [inline reports/m1_proposal.md:9]: I added more general stakeholders as well, based on the personas in section 3. Feel free to reword it!

```suggestion
Possible users and stakeholders who would find value in our dashboard include governments, policy makers, scientists, environmental advocacy organizations, as well as individuals who might be interested in learning more about the impact of CO2 emission across the globe. Please find more detailed discussion of the use cases in section 3.
```

**Review by monazhu — COMMENTED** [review]
Made a few minor suggestions to try and tie the different sections together.

**Comment by Rachel0619** [comment]
Thanks for reviewing. Changes accepted and committed!

**Review by monazhu — APPROVED** [review]
LGTM!!

**Pull request merged**